opa/models/session.py

```diff
@@ -0,0 +1,65 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class Session(models.Model):
+    _name = 'session'
+    _description = 'Session from an Open Academy course'
+    _rec_name = 'title'
+
+    title = fields.Char(required=True)
+    starts_on = fields.Date(required=True)
+    ends_on = fields.Date(default=fields.Date.today)
+    duration = fields.Float(string='Duration (hours)')
+    seats = fields.Integer()
+    course_id = fields.Many2one(comodel_name='course', required=True)
+    instructor_id = fields.Many2one(comodel_name='res.partner',
```

Comment:
Break after parenthesis

Reply:
@antonag32 apply the Black formatter suggestion:

```suggestion
    instructor_id = fields.Many2one(
        comodel_name='res.partner',
        domain=['|', ('instructor', '=', True), ('category_id.parent_id.name', '=', 'Teacher')],
    )
```

Reply:
Done